sql/report_queries/erm/costs/erm_inventory_cost_trans.sql

```diff
@@ -0,0 +1,180 @@
+/** Documentation of ERM INVENTORY COUNT QUERY
+
+DERIVED TABLES
+folio_reporting.invoice_adjustments_ext
+folio_reporting.po_lines_phys_mat_type
+folio_reporting.po_lines_er_mat_type
+folio_reporting.invoice_lines_adjustments
+folio_reporting.instance_subjects
+folio_reporting.instance_formats
+folio_reporting.instance_ext
+folio_reporting.holdings_ext
+folio_reporting.locations_libraries
+folio_reporting.transaction_invoices
+
+TABLES
+invoice_invoices
+invoice_lines
+po_lines
+
+FILTERS: FOR USERS TO SELECT
+payment_date , invoice_line_status, po_line_order_format, instance_subject, mode_of_issuance_name,
+format_name, library_name
+
+ */
+WITH parameters AS (
+    SELECT
+        -- filters on invoice level
+        -- Please comment/uncomment one pair the these parameters if you want to define the date range of paid invoices
+        NULL::DATE AS start_date,
+        NULL::DATE AS end_date,
+        --'2021-01-01' :: DATE AS start_date, -- start date day is included in interval
+        --'2022-01-01' :: DATE AS end_date, -- end date day is NOT included in interval -> enter next day
+        -- filters on invoice line level
+        'Paid'::VARCHAR AS invoice_line_status, -- Enter your invoice line status eg. 'Paid' or 'Approved'etc.
+        -- filters on purchase order line level
+        ''::VARCHAR AS po_line_order_format, -- Enter your po line format eg. 'Electronic Resource', 'Physical Resource', 'P/E Mix' etc.
+        -- filters on instance level
+        ''::VARCHAR AS instance_subject, -- Enter your  eg.  etc.
```

Comment:
Line 38, 39, 41:
Should there be an example in the comments for filters?

Reply:
Have added them now.